src/components/SearchBar/SearchBar.js

```diff
@@ -6,14 +6,14 @@ import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
 import 'url-search-params-polyfill'
 
 /**
- * Component that renders the searchBar component.
+ * Component that renders the searchBar component, which displays a search bar which can be used within a component/page.
  *
  * @component
  *
- * @param {string} width - Parameter that is used to set the width of an element within the rendered component.
- * @param {string} exactWidth - Parameter that can be set as a value for the width parameter.
- * @param {boolean} compInNavigation - Parameter that is used in a if statement to set values to certain variables.
- * @param {string} placeholder - Parameter that shows a placeholder in the SearchBar.
+ * @param {string} width - Parameter that contains the width of the SearchBar component.
+ * @param {string} exactWidth - Parameter that contains the exact width of the SearchBar component.
+ * @param {boolean} compInNavigation - Parameter that is used to check if the component is not in the navbar.
```

Comment:
`compInNavigation - Parameter that is used to check if the component is not in the navbar.`

-> remove the `not`

`compInNavigation - Parameter that is used to check if the component is in the navbar.`